packages/react-components/react-combobox/etc/react-combobox.api.md

```diff
@@ -124,7 +124,7 @@ export type ListboxSlots = {
 };
 
 // @public
-export type ListboxState = ComponentState<ListboxSlots> & OptionCollectionState & SelectionState & {
+export type ListboxState = ComponentState<ListboxSlots> & OptionCollectionState & Pick<SelectionProps, 'multiselect'> & SelectionState & {
```

Comment:
With the change to `SelectionState`, this is effectively no change to `ListboxState`